integration-tests/gradle/projects/it-js-ir-0/src/main/kotlin/RootPackageClass.kt

```diff
@@ -12,7 +11,8 @@ class RootPackageClass {
     val description = "I do live in the root package!"
 }
 
-fun RComponent<*, *>.params() = URLSearchParams()
+// sample method that uses classes from dom and react, should compile
+fun URLSearchParams.react(props: Props, state: State) {}
```

Comment:
RComponent existed in 1.4, but seems to have been removed in 1.5 or 1.6. Moreover, `Props` and `State` were also moved from `kotlin-react-router-dom` to `kotlin-react` in 1.5. This makes it difficult to support testing of 1.4. 

Since 1.4 is already deprectaed, I decided to drop it and make sure it works for 1.5 and 1.6. Hopefully, we'll drop 1.4 generally when 1.7 comes out, so not a big deal in my opinion.